### ISSUE TITLE
Add support for Vorapis/2014 Google Player?

### ISSUE THREAD

**Issue by pumpkintheproot** [web-scrobbler/web-scrobbler#5026]
**Is your feature request related to a problem? Please describe.**
Partially, because with Project Vorapis (https://github.com/vorapis-yt/v3, basically a script that brings back the old look of YouTube) uses a 2014 google player (which, in case turn, does not seem to work with web scrobbler)
![firefox_BKXB8R6v9R](https://github.com/user-attachments/assets/787de907-0ac2-4c56-8c26-857804efda72)

**Describe the solution you'd like**
Basically, support for the 2014 google player so Web Scrobbler can pick up these songs

**Describe alternatives you've considered**
Not using the script/web scrobbler

**Additional context**
https://github.com/vorapis-yt/v3/issues/24
